### DISCUSSION TITLE
Incorrect results for Digital Ocean and MS Azure ?

### DISCUSSION THREAD

**Discussion by dosssman** [VarunAgw/CloudPingTest#13] [Q&A] [answered]
Greetings.

Thank you very much for your hard work in creating and maintaining this server latency benchmarking tools.
I was recently checking which server might fit my needs, and while the [Vultr](https://cloudpingtest.com/vultr) variant works relatively well, either [Digital Ocean](https://cloudpingtest.com/digital_ocean) or [MS Azure](https://cloudpingtest.com/azure) show some unrealistic unrealistic results.
All location have very close latency values, and do not match physical reality:

![image](https://github.com/VarunAgw/CloudPingTest/assets/21224272/988db2a2-1fcf-4a3d-9617-f704cf973edb)

Do you happen to have any idea about why that would be ?

Thank again for your time.
Looking forward to hear from you.

**Comment by VarunAgw** [comment]
You are right. It's completely broken because of some API change. I will try to work on it when I get a chance

**Comment by phillip-haydon** [comment]
It seems broken for AWS too, from Taiwan to Oregon on my API I get 150ms, but from your ping test I get 400ms. Something isn't right. :(

**Comment by VarunAgw** [comment, marked as answer]
I have removed Digital Ocean (unfortunately, it's not possible to test it anymore).

Azure is fixed.
